admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -104,6 +110,50 @@ export class AlertBannerModel extends HoistModel {
         this.refreshAsync();
     }
 
+    loadPreset(preset: PlainObject) {
+        this.formModel.setValues({...preset, expires: null});
+    }
+
+    @action
+    addPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values;
+        const dateCreated = fmtDateTime(Date.now());
+        this.savedPresets = sortBy(
+            [...this.savedPresets, {message, intent, iconName, enableClose, dateCreated}],
+            [preset => preset.intent, preset => preset.message]
+        );
+        this.savePresetsAsync();
+    }
+
+    @action
+    removePreset(preset: PlainObject) {
+        this.savedPresets = without(this.savedPresets, preset);
+        this.savePresetsAsync();
```

Comment:
Let's add a confirm here, so a user who accidentally clicks the (x) button doesn't immediately delete something that they can't get back.

The best way to do that is to make the overall method async, then at the top `await XH.confirm({...})` - if the value is false, return without modifying the presets. (Sub in an appropriate message / title for the confirm dialog)